coffee-chats/src/main/java/com/google/step/coffee/servlets/GroupInfoServlet.java

```diff
@@ -4,16 +4,40 @@
 import com.google.step.coffee.JsonServlet;
 import com.google.step.coffee.JsonServletRequest;
 import com.google.step.coffee.PermissionChecker;
+import com.google.step.coffee.data.GroupStore;
 import com.google.step.coffee.entity.Group;
 
 import javax.servlet.annotation.WebServlet;
 import java.io.IOException;
 
 @WebServlet("/api/groupInfo")
 public class GroupInfoServlet extends JsonServlet {
-  @Override
-  public Object get(JsonServletRequest request) throws IOException, HttpError {
+  private GroupStore groupStore = new GroupStore();
+
+  private Group getGroup(JsonServletRequest request) throws HttpError {
+    // TODO(tsarn): also check that the user has write access to the group
```

Comment:
Why should getting information about the group require write access?

Reply:
I have already done that todo in another branch, so it doesn't really matter

Reply:
You didn't answer my question

Reply:
Yeah, it shouldn't, I have already fixed it already, don't know what else to say

Reply:
From your previous comment I understood that you already added this check.

How am I supposed to guess that what you did is to remove it instead? (or did you?) You didn't link to that change